tests/python/unittest/test_operator.py

```diff
@@ -6496,6 +6496,75 @@ def test_softmax():
     check_smoothed_softmax_grad(default_context())
 
 
+@with_seed()
+def test_SoftmaxOutput_normalization():
```

Comment:
Thanks for the quick fix! 
nit: could you change `test_SoftmaxOutput_normalization` to `test_softmax_output_normalization `?

Reply:
I will change it later. Thank you!